bitwarden_license/test/Commercial.Core.Test/SecretManagerFeatures/AccessPolicies/CreateAccessPoliciesCommandTests.cs

```diff
@@ -51,4 +51,36 @@ await sutProvider.GetDependency<IAccessPolicyRepository>().Received(1)
 
         await sutProvider.GetDependency<IAccessPolicyRepository>().DidNotReceiveWithAnyArgs().CreateManyAsync(default);
     }
+
+
+    [Theory]
+    [BitAutoData]
+    public async Task CreateAsync_NotUnique_ThrowsException(
```

Comment:
Could we expand this with tests for each access policy? To ensure we better catch any potential regression in this.

Reply:
Resolved in https://github.com/bitwarden/server/pull/2507/commits/22ca6cbdc2eff07ce0c671c8639a3b4c78877067